dev-infrastructure/modules/aks-cluster-base.bicep

```diff
@@ -125,7 +138,7 @@ resource aks_keyvault_crypto_user 'Microsoft.Authorization/roleAssignments@2022-
   }
 }
 
-resource vnet 'Microsoft.Network/virtualNetworks@2023-09-01' = {
+resource vnet 'Microsoft.Network/virtualNetworks@2023-11-01' = {
```

Comment:
This allows vnet/subnet to behave idempotently now from https://github.com/Azure/azure-quickstart-templates/issues/2786